sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/AbstractScheduler.java

```diff
@@ -53,6 +54,11 @@
     private final Object inProgressLock = new Object();
     private final Set<Protos.OfferID> offersInProgress = new HashSet<>();
 
+    // Metrics
+    private final Counter receivedOffers = SchedulerUtils.getMetricRegistry().counter("received_offers");
```

Comment:
`offer.input.received_offers`? Not sure about the names but it feels like all the stuff relating to offer input/output could be grouped together.